tests/postgres_tests/test_indexes.py

```diff
@@ -79,7 +79,13 @@ def test_deconstruction(self):
         path, args, kwargs = index.deconstruct()
         self.assertEqual(path, 'django.contrib.postgres.indexes.GinIndex')
         self.assertEqual(args, ())
-        self.assertEqual(kwargs, {'fields': ['title'], 'name': 'test_title_gin'})
+        self.assertEqual(kwargs, {'fields': ['title'], 'name': 'test_title_gin',
+                                  'fastupdate': None, 'gin_pending_list_limit': None})
+
+    def test_illegal_list_limit(self):
+        with self.assertRaisesMessage(ValueError, 'gin_pending_list_limit must be between {} and {}'.format(
```

Comment:
The documentation just has `gin_pending_list_limit (integer)` (also in the dev version of the docs), so most likely that it's fixed to limits of the C standard. In `GinIndex`, is it fine to leave the constants, or would it also make sense to replace them with hardcoded values?